### DISCUSSION TITLE
Suggestion (that might be rejected): Scratch 3.0 → 2.0

### DISCUSSION THREAD

**Discussion by DatGamerYoloYT** [ScratchAddons/ScratchAddons#4659] [Ideas]
Basically idk how to explain this but it's just a reverse version of the Scratch 2.0 → 3.0 addon where Scratch 3.0-styled pages look like scratch 2.0 (basically also helps fix the inconsistency but by having it be scratch 2.0 instead of all 3.0)

might be rejected, idk but it's prob likely to be rejected due to the idea ¯\_(ツ)_/¯

if it's rejected i won't care that much as it's just a reversing of the Scratch 2.0 → 3.0 addon

**Comment by Samq64** [comment]
Duplicate of #985.

**Reply by DatGamerYoloYT** [reply]
bruh wtf why didn't i think before i post

**Reply by DatGamerYoloYT** [reply]
delete this pls

**Reply by WorldLanguages** [reply]
No worries
